### PR TITLE
Try sleep until wakeup 🧼

### PR DESCRIPTION
**Pull request by antaranyan**
⏰

### CODE REVIEW

**Review by hisusqristos — APPROVED** [review]
nice

File: src/sleep.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 // (*ᴗ͈ˬᴗ͈)ꕤ*.ﾟ♡ sweeet dreams
 
 const sleep = (milliseconds: number) => {
-  const date = Date.now();
-  let currentDate = null;
+  const wakeUpTime = Date.now() + milliseconds;
   do {
-    currentDate = Date.now();
-  } while (currentDate - date < milliseconds);
+    // ꒰ ᵕ༚ᵕ꒱
```
Comment (antaranyan) [inline src/sleep.ts]:
```suggestion
    //ᵕ༚ᵕ
```

File: src/sleep.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 const sleep = (milliseconds: number) => {
   const wakeUpTime = Date.now() + milliseconds;
   do {
-    // ꒰ ᵕ༚ᵕ꒱
+    //ᵕ༚ᵕ
```
Comment (hisusqristos) [inline src/sleep.ts]: ( ु⁎ᴗ_ᴗ⁎)ु.｡zZ

Comment (antaranyan) [reply src/sleep.ts]:
```suggestion
    // ( ु⁎ᴗ_ᴗ⁎)ु.｡zZ
```

hm?

Comment (hisusqristos) [reply src/sleep.ts]: da

**Pull request closed without merging**